tests/services/test_record_service_tasks.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020-2021 CERN.
+# Copyright (C) 2020-2021 Northwestern University.
+#
+# Invenio-Drafts-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Service tests.
+
+Test to add:
+- Read a tombstone page
+- Read with missing permissions
+- Read with missing pid
+"""
+
+from io import BytesIO
+
+import pytest
+from invenio_db import db
+from mock_module.service import ServiceConfig
+
+from invenio_drafts_resources.services import RecordService
+from invenio_drafts_resources.services.records.tasks import cleanup_drafts
+
+
+#
+# Fixtures
+#
+@pytest.fixture()
+def input_data(input_data):
+    """Enable files."""
+    input_data['files']['enabled'] = True
+    return input_data
+
+
+@pytest.fixture()
+def service(appctx, location, file_service, draft_file_service):
+    """Service instance."""
+    return RecordService(
+        ServiceConfig,
+        files_service=file_service,
+        draft_files_service=draft_file_service
+    )
+
+
+def add_file_to_draft(draft_file_service, draft_id, file_id, identity):
+    """Add a file to the record."""
+    draft_file_service.init_files(draft_id, identity, data=[{'key': file_id}])
+    draft_file_service.set_file_content(
+        draft_id, file_id, identity, BytesIO(b'test file content')
+    )
+    result = draft_file_service.commit_file(draft_id, file_id, identity)
+    return result
+
+
+def test_hard_delete_soft_deleted_task(
+        app, service, identity_simple, input_data
+        ):
+    draft = service.create(identity_simple, input_data)
+    add_file_to_draft(service.draft_files, draft.id, 'test', identity_simple)
+    record = service.publish(draft.id, identity_simple)
+    draft_model = service.draft_cls.model_cls
+
+    assert len(draft_model.query.filter(
+        draft_model.is_deleted == True  # noqa
+    ).all()) == 1
+    cleanup_drafts(current_service_imp=service, seconds=0)
+    db.session.commit()
```

Comment:
major: why is this commit needed?

This test is calling the task, which calls the service, which is commiting. See [here](https://github.com/inveniosoftware/invenio-drafts-resources/pull/145/files#diff-dd63614caabee35c446c8b77120519bfdd2dc6daf100416ed6469760b5d2f0a2R515)